editoast/migrations/2024-02-21-002011_create_v2_scenario_trainschedule_timetable/up.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE timetablev2 (
+    id int8 PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    electrical_profile_set_id int8 NULL REFERENCES electrical_profile_set(id) ON DELETE CASCADE
+);
+CREATE TABLE scenariov2 (
+    id int8 PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    infra_id int8 NOT NULL REFERENCES infra(id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED,
+    name varchar(128) NOT NULL,
+    description varchar(1024) NOT NULL,
+    creation_date timestamptz NOT NULL,
+    last_modification timestamptz NOT NULL,
+    tags text [] NOT NULL,
+    timetable_id int8 NOT NULL UNIQUE REFERENCES timetablev2(id) DEFERRABLE INITIALLY DEFERRED,
+    study_id int8 NOT NULL REFERENCES study(id) ON DELETE CASCADE
+);
+CREATE TABLE trainschedulev2 (
```

Comment:
We should use a snake case to be consistent.

```suggestion
CREATE TABLE train_schedule_v2 (
```